tensorzero-core/src/db/clickhouse/select_queries.rs

```diff
@@ -337,4 +337,84 @@ impl SelectQueries for ClickHouseConnectionInfo {
                 })
             })
     }
+
+    async fn get_feedback_timeseries(
+        &self,
+        function_name: String,
+        metric_name: String,
+        variant_names: Option<Vec<String>>,
+        interval_minutes: u32,
```

Comment:
there is an enum for granularity somewhere-- we should use that -> then we should use clickhouse INTERVAL math for our time calculations, because a month is not a fixed number of minutes and in edge cases neither is a week or a day.